### ISSUE TITLE
VRPN related compile error: union wait, incomplete type

### ISSUE THREAD

**Issue by emiax** [opensgct/sgct#13]
When compiling sgct on Ubuntu 17.04 with gcc 6.3.0, the following error is generated:

```
sgct/src/deps/vrpn/vrpn_Connection.C: In function ‘int vrpn_start_server(const char*, char*, char*, const char*)’:
sgct/src/deps/vrpn/vrpn_Connection.C:2492:24: error: aggregate ‘vrpn_start_server(const char*, char*, char*, const char*)::wait status’ has incomplete type and cannot be defined
             union wait status;
```

Changing `union wait` to `int` fixes the problem. This seems to be fixed in vrpn since https://github.com/vrpn/vrpn/commit/70d47fec7e3be85f5f05c8ac5be99dacbd8be257. What are the routines for updating vrpn and other deps of sgct? Can we make it into a git submodule?

**Comment by einsweniger** [comment]
Can confirm: this still happens with gcc 7.2.0.
The fix is the same as for OP.

**Comment by alexanderbock** [comment]
Yes, I think we should change all the dependencies into a submodule ...  let's put it on the big ol' pile of refactoring.
